lib/Network/Wai/EventSource/Streaming.hs

```diff
@@ -26,7 +27,11 @@ import qualified Data.Attoparsec.ByteString.Streaming as SA
 import           Data.Binary.Builder (Builder, fromByteString, toLazyByteString)
 import qualified Data.ByteString as B
 import qualified Data.ByteString.Lazy as BL
+#if MIN_VERSION_streaming_bytestring(0,1,7)
+import qualified Streaming.ByteString.Char8 as SB
```

Comment:
`Data.ByteString.Streaming.Char8` has been marked as deprecated in streaming-bytestring 0.1.7